test/intl402/Temporal/Instant/formatrange-fails.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant
```

Comment:
This should be a different ID, since we are testing functionality from Intl.DateTimeFormat.prototype.formatRange. (Although I'm not sure if it should be the ID from https://tc39.es/ecma402/#sec-intl.datetimeformat.prototype.formatRange where the functionality will eventually be described, or it should have a dummy ID for now)

The test file should also be moved to `test/intl402/Intl/DateTimeFormat/prototype/formatRange/` and it should have a more descriptive name like `arguments-different-types.js`.

Reply:
done (I settled on `sec-intl.datetimeformat.prototype.formatRange` for the ID, though if that needs to be changed, we can change it).